command/loadbot/execution.go

```diff
@@ -0,0 +1,224 @@
+package loadbot
+
+import (
+	"context"
+	"fmt"
+	"github.com/0xPolygon/polygon-sdk/helper/tests"
+	txpoolOp "github.com/0xPolygon/polygon-sdk/txpool/proto"
+	"github.com/0xPolygon/polygon-sdk/types"
+	"github.com/umbracle/go-web3"
+	"github.com/umbracle/go-web3/jsonrpc"
+	"google.golang.org/grpc"
+	empty "google.golang.org/protobuf/types/known/emptypb"
+	"math/big"
+	"sync"
+	"time"
+)
+
+// Configuration represents the loadbot run configuration.
+// It contains the required parameters to run the stress test.
+type Configuration struct {
+	TPS       uint64          // Number of transactions per second
+	Value     *big.Int        // The value sent in each transaction
+	Gas       uint64          // The transaction's gas
+	GasPrice  *big.Int        // The transaction's gas price
+	Accounts  []types.Address // All the accounts used by the bot to send transaction
+	RPCURLs   []string        // The JSON RPC endpoints, used to send transactions
+	ChainID   uint64
+	TxnToSend uint64 // The number of transaction to send
+	GRPCUrl   string // The gRPC url, used while verifying transactions to check if the TxPool is empty
+}
+
+type Metrics struct {
+	m         sync.Mutex
+	Total     uint64        // The total number of transactions processed
+	Failed    uint64        // The number of failed transactions
+	Duration  time.Duration // The execution time of the loadbot
+	TxnHashes []web3.Hash   // The hashes of the transactions sent
+}
+
+// createClients will create JSON RPC clients using the provided addresses in the CLI.
+// Each one of these clients will send transaction(s), each one after another.
+func (c *Configuration) createClients() ([]*jsonrpc.Client, error) {
+	var clients []*jsonrpc.Client
+
+	for _, url := range c.RPCURLs {
+		conn, err := jsonrpc.NewClient(fmt.Sprintf("http://%s", url))
+		if err != nil {
+			return nil, fmt.Errorf("failed to connect to server: %v", err)
+		}
+		clients = append(clients, conn)
+	}
+	return clients, nil
+}
+
+// We create the transactions using createTransactionObjects before the loadbot send them.
+func (c *Configuration) createTransactionObjects() ([]*web3.Transaction, error) {
+	var transactions []*web3.Transaction
+	var nonces = make(map[types.Address]uint64)
+	var numberOfAccounts = uint64(len(c.Accounts))
+
+	// Initialize accounts nonces at 0.
+	for _, account := range c.Accounts {
+		nonces[account] = 0
+	}
```

Comment:
FYI, you don't have to initialize map every time. map returns zero-value for non-existing key.

```go
m := make(map[string]int)

// map access for existing key
m["foo"] = 10
fmt.Println(m["foo"]) // => 10
x, ok := m["foo"]
fmt.Println(x, ok) // => 10, true

// map access for non-existing key
fmt.Println(m["bar"]) // => 0
x, ok = m["bar"]
fmt.Println(x, ok) // => 0, false
```